suite-native/alerts/src/useAlert.ts

```diff
@@ -1,10 +1,14 @@
 import { useSetAtom } from 'jotai';
 
-import { hideAlertAtom, showAlertAtom } from './alertsAtoms';
+import { AlertType, hideAlertAtom, showAlertAtom } from './alertsAtoms';
 
 export const useAlert = () => {
     const showAlert = useSetAtom(showAlertAtom);
-    const hideAlert = useSetAtom(hideAlertAtom);
+    const hide = useSetAtom(hideAlertAtom);
+
+    const hideAlert = (type?: AlertType) => {
+        hide(type);
+    };
 
     return { showAlert, hideAlert };
```

Comment:
Why is this not possible? 🤔 
```suggestion
    const hideAlert = useSetAtom(hideAlertAtom);

return { showAlert, hideAlert }
```

Reply:
https://github.com/trezor/trezor-suite/commit/9d78fd8d18ba087bdf3a63a3ff2a7094dfb27a09, leftover from earlier implementation

Reply:
good catch